### PR TITLE
Update recipe `bleties`

### PR DESCRIPTION
**Pull request by kbseah**
From version 5, Muscle has changed the format of its version number reporting. This conflicts with the regex used in Bleties to check dependencies. However version 3.8.1551 is adequate for BleTIES, so fixing dependency to this specific version.

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by kbseah** [comment]
@BiocondaBot please add label

File: recipes/bleties/meta.yaml [hunk 1]
```diff
@@ -27,7 +27,7 @@ requirements:
     - htslib=1.9
     - biopython=1.76
     - scipy=1.5.0
-    - muscle>=3.8.1551
+    - muscle=3.8.1551
```
Comment (bgruening) [inline recipes/bleties/meta.yaml:30]: shouldn't this be `<=`

Comment (kbseah) [reply recipes/bleties/meta.yaml:30]: No, 3.8.1551 was the version used for testing. Lower versions were not tested. Higher versions break version check as described above.

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
@BiocondaBot please merge

**Comment by kbseah** [comment]
thank you @bgruening!

**Pull request merged**